frontend/src/pages/chat/components/chat-layout/chat-layout.tsx

```diff
@@ -86,9 +92,17 @@ const ChatLayout: React.FC<Properties> = ({ filter }) => {
               />
             );
           })}
+        {isChatbotReplyLoading && (
+          <div>
```

Comment:
do we need this div around loader?

Reply:
unfortunately, we do. Without this div, the dots will squeeze. I can move it to a separate component if you want